test/prettier.mjs

```diff
@@ -1,15 +1,13 @@
 import prettier from 'prettier';
 
-let prettierOptions = undefined;
-
 /**
  * Use Prettier to format content.
  * Cache PrettierOptions in a singleton.
  *
  * @param {string} content
  * @return {string}
  */
-export async function formatter(content) {
+export async function formatter(content, prettierOptions) {
```

Comment:
This change moves away from a singleton, intended?

Reply:
Yes, it was intended in order to allow for parameterized options in test, where the json parser is used in some cases. But if there is some other side effect of the change, that may not have been intended. I see that I forgot to remove line 5 at the very least, which is now a stale comment.